README.md

```diff
@@ -28,6 +28,43 @@ export RACKSPACE_APIKEY=...
 docker-compose build && docker-compose up -d
 ```
 
+Alternatively, you can manually run each docker container with:
+
+```bash
+# startup content service dependencies
+docker run -d --name elasticsearch elasticsearch:1.7
+docker run -d --name mongo mongo:2.6
+
+# build and deploy the content service
+cd {wherever you have the deconst/content-service}
+docker build --tag content-service:1.0.0 .
+docker run -d -p 9000:8080 \
+              -e NODE_ENV=development \
+              -e STORAGE=memory \
+              -e MONGODB_URL=mongodb://mongo:27017/content \
+              -e ELASTICSEARCH_HOST=http://elasticsearch:9200/ \
+              -e ADMIN_APIKEY=$RACKSPACE_APIKEY \
```

Comment:
This isn't used as a Rackspace credential - it's an API key that the content service will _recognize_, not one that it _authenticates with_. I usually generate a random string using something like:

```bash
hexdump -v -e '1/1 "%.2x"' -n 128 /dev/random
```